source/driver/Driver.cpp

```diff
@@ -612,6 +612,12 @@ bool Driver::parseAllSources() {
                         missing.emplace(name);
                 }
             }
+
+            for (auto bd : meta.interfacePorts) {
```

Comment:
What does 'bd' stand for here? Can you either use 'ip' for interface port or use something more descriptive like 'portSyntax' or whatever.